caster-front/src/graphql.ts

```diff
@@ -1,32 +1,32 @@
-import gql from "graphql-tag";
-import * as Urql from "@urql/vue";
+import gql from 'graphql-tag';
```

Comment:
Do we have single or double tics?

Reply:
I don't know? :/ This was linted w/o vscode but through `npm`, so maybe we have some different settings in vscode vs using npm?